library/src/scripts/forms/select/SelectBox.tsx

```diff
@@ -84,21 +83,23 @@ export default class SelectBox extends React.Component<ISelfLabelledProps | IExt
 
     public render() {
         const classes = selectBoxClasses();
+
         const classesDropDown = dropDownClasses();
-        const classesMetas = metasClasses();
         const selectItems = this.props.children.map((child, i) => {
             const selected = this.state.selectedIndex === i;
             return (
                 <DropDownItemButton
                     key={this.props.id + "-item" + i}
                     className={classNames({ isSelected: child.selected })}
                     name={child.name}
```

Comment:
* Either use the full name of the language, or don't put a title at all
* Don't put an aria label on these. They're self explanatory if they are links with the name of the locale in there.